projects/fsastorefrontlib/src/core/checkout/services/category/category.service.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { TestBed } from '@angular/core/testing';
+import { CategoryService } from './category.service';
+
+const category = 'testCategory';
+
+describe('CategoryService', () => {
+  let service: CategoryService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [CategoryService],
+    });
+    service = TestBed.inject(CategoryService);
+  });
+
+  it('should be created', () => {
+    expect(service).toBeTruthy();
+  });
+
+  it('should set active category', () => {
+    let categoryCode;
+    service.setActiveCategory(category);
+    service.getActiveCategory().subscribe(data => (categoryCode = data));
```

Comment:
Unsubscribe is missing, this can cause random failures of other tests.
My proposal:
```
    service.getActiveCategory().subscribe(data => {
      data === category;
    }).unsubscribe()
```